### PR TITLE
Document how to test an alias

### PR DESCRIPTION
**Pull request by jdbaldry**
Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

File: docs/sources/writing-guide/front-matter/index.md [hunk 1]
```diff
@@ -211,3 +211,31 @@ Aliases can be relative or absolute paths.
 Do not include an `aliases` entry that refers to the initial published website directory.
 The version in the URL path can cause undesirable redirects, such as a redirect from latest content to an old version.
 Aliases should be relative and not absolute paths so that old versions do not steal redirects from "latest" content when it is moved around.
+
+### Test an alias
+
+To test an alias results in the correct redirect, use your browser or a command-line tool for making HTTP requests.
+
+#### Using the browser
```
Comment (brendamuir) [inline docs/sources/writing-guide/front-matter/index.md:219]:
```suggestion
#### Use the browser
```

File: docs/sources/writing-guide/front-matter/index.md [hunk 2]
```diff
@@ -211,3 +211,31 @@ Aliases can be relative or absolute paths.
...
+
+1. Start the documentation webserver with `make docs`.
+1. Browse to the URL of the page that should be redirected.
+1. Confirm that you are redirected to the desired page.
+
+#### Using `cURL`
```
Comment (brendamuir) [inline docs/sources/writing-guide/front-matter/index.md:225]:
```suggestion
#### Use `cURL`
```

**Review by brendamuir — APPROVED** [review]

File: docs/sources/writing-guide/front-matter/index.md [hunk 3]
```diff
@@ -211,3 +211,31 @@ Aliases can be relative or absolute paths.
...
+
+### Test an alias
+
+To test an alias results in the correct redirect, use your browser or a command-line tool for making HTTP requests.
+
+#### Using the browser
+
+1. Start the documentation webserver with `make docs`.
+1. Browse to the URL of the page that should be redirected.
+1. Confirm that you are redirected to the desired page.
```
Comment (imatwawana) [inline docs/sources/writing-guide/front-matter/index.md:223]: Suggest adding an example
```suggestion
1. Confirm that you are redirected to the desired page.

For example, if you want the page `https://grafana.com/docs/grafana/latest/panels/working-with-panels/` to redirect to `https://grafana.com/docs/grafana/latest/panels-visualizations/panel-editor-overview/`, navigate to the URL in the browser to confirm the redirect is working: 

```
http://localhost:3002/docs/grafana/latest/panels/working-with-panels/ 
```
```

Comment (jdbaldry) [reply docs/sources/writing-guide/front-matter/index.md:223]: Thank you, added in 2d8b164

File: docs/sources/writing-guide/front-matter/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/writing-guide/front-matter/index.md]: Thank you, fixed in a48dd00

**Pull request merged**